docs/data-structures/Singly-Linked-list.mdx

```diff
@@ -0,0 +1,589 @@
+---
+title: Singly Linked list
+slug: /data-structures/Singly-Linked-list
+---
+
+## Introduction
+Singly Linked list is a type of linear Linked list having collection of variable number of data items called nodes. Each node has two fields - one is the **information field** containing the data value of that node, and the other is the **link field** which contains the memory address of the next node. Since here we have only one link field in each node, it is termed as Singly Linked list.
+
+![Capture](https://user-images.githubusercontent.com/70858557/114307192-2bab1300-9afc-11eb-98cd-cd13459394fd.PNG)
```

Comment:
Did you make this image?

Reply:
> Did you make this image?

Yes, I made the image.

Reply:
That's awesome. We have published a lot of articles in past few days. If you would like to make images on them, please feel free to make an issue.

Reply:
> That's awesome. We have published a lot of articles in past few days. If you would like to make images on them, please feel free to make an issue.

I would love to. I will make issues for them soon.